src/App.js

```diff
@@ -1,24 +1,89 @@
-import React, { Component } from 'react';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "./grid.css";
+import weather from "./data/weather.json";
+import Search from "./Components/Search";
+import CurrentWeather from "./Components/CurrentWeather";
+import Upcoming from "./Components/Upcoming";
 
 class App extends Component {
-
   constructor(props) {
     super(props);
     this.state = {
+      weather: weather,
+      city: "London",
+      err: ""
     };
   }
+  //json data^
+  componentDidMount() {
+    this.handleClick();
+  }
+
+  changeHandler = event => {
+    this.setState({
+      city: event.target.value
+    });
+  };
+
+  handleClick = () => {
+    console.log("i've been clicked");
+    fetch(
+      `http://api.openweathermap.org/data/2.5/forecast?q=${
+        this.state.city
+      }&cnt=8&units=metric&appid=${"a75d5e4c8cb937be92dab81195a1c637"}`
```

Comment:
Nice one getting this to work.
The idea behind using template strings (the back ticks) + the `${someVariable}` syntax is when you want to insert Javascript variables inside a string. With your API key, it's already a string and you're inserting it inside a string which is overkill. The idea would be that you have saved your API key inside a variable (usually in a file called `.env`), and you import it into your API call string using that variable name.
So it would look something like this:
`http://api.openweathermap.org/data/2.5/forecast?q=${
        this.state.city
      }&cnt=8&units=metric&appid=${APIKey}`